tempto-core/src/main/java/com/teradata/tempto/internal/query/QueryExecutorModuleProvider.java

```diff
@@ -54,7 +54,9 @@ protected void configure()
                 bind(JdbcConnectionsPool.class).toInstance(jdbcConnectionsPool);
                 Set<String> definedJdcbConnectionNames = jdbcConnectionsConfiguration.getDefinedJdcbConnectionNames();
                 for (String connectionName : definedJdcbConnectionNames) {
-                    bindDatabaseConnectionBeans(connectionName);
+                    if (!connectionName.equals("cassandra")) {
```

Comment:
I don't necessarily agree here.
Cassandra has nothing to do with JDBC so I don't think we should hardcode exception for it inside JDBC "internal" classes. It's hacked on the quite generic level.
I agree we should not be binding `QueryExecutor` if it's not exposed nor needed but I claim it's outside of scope of this PR.